### ISSUE TITLE
Add 'How to Redeem' section to the offer profile pages

### ISSUE THREAD

**Issue by tallmadge** [SmallShopsUnited/React-Native-Mobile-App#11]
Bryce is adding a new field to the Gift Pack Offers called 'How to Redeem'.  It's a text field that will allow me to add notes to each offer instructing the user how to redeem this offer.

Beneath the "Call" and "Get Directions" sections, the format should look as follows depending on the offer type (promo code vs. in-store):

**In-Store Offers:**
Gift Pack Offer
Conditions
How to Redeem

**Promo Code Offers:**
Gift Pack Offer
Conditions
How to Redeem
Your Unique Promotional Code
[code]
Copy to Clipboard
Go To Purchase

**Comment by tallmadge** [comment]
I'm going to wait until we have actual data being pulled into that section before closing this issue.

**Comment by tallmadge** [comment]
I don't think this info is getting pulled in when I pull down on the index page to refresh the offers.  For example, there was no info in the How to Redeem section for a particular offer when I was looking at it in the app.  I then went into my admin to add both the start & end dates for the offer, and the How to Redeem info for the offer.  I then went back to the Gift Pack index page and pulled down to refresh the offers.  I went back into this specific offer, but the data had not been added.  It wasn't until I uninstalled and reinstalled the app that the data got pulled in.
